app/views/users/index.html.erb

```diff
@@ -1,19 +1,19 @@
-<div class="container">
-    <div class="row">
-
-        <div class="col-md-12 d-flex ">
-
-            <img src="" height="50">
-
-            <div class="">
-
-                <h3>My name</h3>
-
-                <span>Number of post: 0</span>
-
+<main>
+  <section>
+    <ul>
+      <% @users.each do |user|%>
+        <%= link_to controller: 'users', action: 'show', id: user.id do %>
+          <li class="card ">
+            <div class="card-content">
+            <div class="user-image border"></div>
```

Comment:
- [ ] Great job working on this milestone. As per the [wireframe](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_root_users.png), you are supposed to display the photo as well. At the moment you are only displaying the user.name and user.posts_counter. Kindly display` user.photo` as well